lib/zendesk_apps_support/assets/templates.js.erb

```diff
@@ -0,0 +1,8 @@
+<% unless templates.nil? %>
+{
+  <% templates.each do |name, fn| %>
+  <%= name %>: <%= fn %>,
```

Comment:
You probably saw, in the PR description, I considered that, as an enhancement, it might be better to precompile the templates out to a file to be cached in S3, rather than precompiling every time.